package.json

```diff
@@ -1,14 +1,13 @@
 {
   "name": "directomatic",
-  "version": "1.0.0",
+  "version": "1.1.0-rc",
   "description": "Redirect Generator and Transformer Robot",
   "main": "dist/worker.js",
   "scripts": {
     "build": "webpack",
     "format": "prettier --write  '*.{json,js}' 'src/**/*.{js,ts}' 'test/**/*.{js,ts}'",
     "lint": "eslint --max-warnings=0 src && prettier --check '*.{json,js}' 'src/**/*.{js,ts}' 'test/**/*.{js,ts}'",
-    "prettier": "prettier --write '*.{json,js}' 'src/**/*.{js,ts}' 'test/**/*.{js,ts}'",
```

Comment:
`npm run prettier` and `npm run format` do the exact same thing, killed the prettier version.